pkgpanda/build/test_util.py

```diff
@@ -0,0 +1,86 @@
+import pkgpanda.build
+
+
+def test_hash_files_in_folder(tmpdir):
+    hash_files_in_folder = pkgpanda.build.hash_files_in_folder
+
+    with tmpdir.as_cwd():
+        # Empty folder
```

Comment:
Ii have disagreements with that FIRST document, haven't read the other one yet. I'd also note it's arguing that you should only test a _single_ behavior inside of a single test.

In this case, we're testing one behavior, and each test adds some new exploration of that feature / corner.

I don't think the code would be better by me copy/pasting or writing more test helper functions to split it into chunks, and it wouldn't produce more useful output to engineers who later work on the code. It would add more volume to the test, and more complexity / pieces which must be simultaneously comprehended to understand the test. It wouldn't make debugging the test particularly simpler / quicker, since the test is already very fast, and it demonstrably builds from one step to the next.